report/https_scan_report.mustache

```diff
@@ -427,10 +437,10 @@ BOD 18-01 includes \href{https://https.cio.gov/guide/#what-about-domains-that-ar
+<<#live_bool>>\color{dhs-blue}\textbf{{<<&domain>><<#ocsp_domain>>*<</oscp_domain>>}} & <<#domain_supports_https_bool>>\checkmark <</domain_supports_https_bool>><<^domain_supports_https_bool>> \xmark <<#https_bad_chain_bool>>\textit{\newline Bad chain}<</https_bad_chain_bool>><<#https_bad_hostname_bool>>\textit{\newline Bad hostname}<</https_bad_hostname_bool>><<#https_expired_cert_bool>>\textit{\newline Expired certificate}<</https_expired_cert_bool>><</domain_supports_https_bool>> & <<#domain_enforces_https_bool>>\checkmark <</domain_enforces_https_bool>><<^domain_enforces_https_bool>>\xmark <</domain_enforces_https_bool>> & <<#domain_uses_strong_hsts_bool>>\checkmark <</domain_uses_strong_hsts_bool>><<^domain_uses_strong_hsts_bool>>\xmark <<#hsts_bool>>\textit{\ \ Low max-age}<</hsts_bool>><</domain_uses_strong_hsts_bool>> & <<^domain_has_weak_crypto_bool>> \checkmark <</domain_has_weak_crypto_bool>><<#domain_has_weak_crypto_bool>> \xmark \textit{\ \ Supports: <<#hosts_with_weak_crypto>><<&weak_crypto_list_str>><</hosts_with_weak_crypto>>} <</domain_has_weak_crypto_bool>> & <<#bod_1801_compliance>>\checkmark <</bod_1801_compliance>><<^bod_1801_compliance>>\xmark <</bod_1801_compliance>> & <<#bod_1801_compliance>><<#hsts_preloaded_bool>> \checkmark <</hsts_preloaded_bool>><<^hsts_preloaded_bool>><<#hsts_preload_pending_bool>> \textit{Pending*} <</hsts_preload_pending_bool>><<^hsts_preload_pending_bool>><<#hsts_preload_ready_bool>> \textit{Preload-ready*} <</hsts_preload_ready_bool>><<^hsts_preload_ready_bool>> \xmark <</hsts_preload_ready_bool>><</hsts_preload_pending_bool>><</hsts_preloaded_bool>><</bod_1801_compliance>><<^bod_1801_compliance>> \xmark <</bod_1801_compliance>> & <<#domain_has_symantec_cert_bool>> \textit{\textbf{Yes}} <</domain_has_symantec_cert_bool>><<^domain_has_symantec_cert_bool>> \textit{No} <</domain_has_symantec_cert_bool>> <</live_bool>>
+<<^live_bool>>\color{dhs-blue}\textbf{{<<&domain>><<#ocsp_domain>>*<</oscp_domain>>}} \textit{\ \ Not Live} & \textit{N/A} & \textit{N/A} & \textit{N/A} & \textit{N/A} & \checkmark & <<#hsts_preloaded_bool>> \checkmark <</hsts_preloaded_bool>><<^hsts_preloaded_bool>><<#hsts_preload_pending_bool>> \textit{Pending*} <</hsts_preload_pending_bool>><<^hsts_preload_pending_bool>><<#hsts_preload_ready_bool>> \textit{Preload-ready*} <</hsts_preload_ready_bool>><<^hsts_preload_ready_bool>> \xmark <</hsts_preload_ready_bool>><</hsts_preload_pending_bool>><</hsts_preloaded_bool>> & \textit{No}<</live_bool>>\\
```

Comment:
Typo here:
```<<#ocsp_domain>>``` should be ```<<#oscp_domain>>```
(o**cs**p vs. o**sc**p)

Reply:
It's the other way around.  `<</oscp_domain>>` should be `<</ocsp_domain>>`.

Reply:
LOL, that's clearly what I meant ;)

Reply:
Thanks for pointing it out.  It would have taken me some time to find that.